idea-plugin/src/com/reason/ide/annotations/ErrorAnnotator.java

```diff
@@ -119,17 +235,121 @@ public void apply(
       }
     }
 
-    problemSolver.reportProblems(file.getVirtualFile(), problems);
+    // Call rincewind on the generated cmt file !
+    ReadAction.run(() -> annotateTypes(project, sourceFile, cmtFile));
+
+    problemSolver.reportProblems(sourceFile, problems);
+  }
+
+  @NotNull
+  public File createTempCompilationDirectory(Project project) {
+    String directoryName = "BS_" + project.getName().replaceAll(" ", "_");
+    try {
+      FileUtil.delete(Paths.get(FileUtil.getTempDirectory(), directoryName));
```

Comment:
do we need to delete it ? if it's already present, it's ok

Reply:
good call